### PR TITLE
fix: task testing documentation

### PR DESCRIPTION
**Pull request by mmalina**
There were a few remaining bits that needed
updating since we stopped using a directory
for each task version.

### CODE REVIEW

**Review by johnbieren — DISMISSED** [review]

**Review by happybhati — DISMISSED** [review]

**Comment by mmalina** [comment]
@johnbieren @happybhati I realized that the first one should actually say "task directories" and not just "tasks", so fixed now. Please check again.

**Review by johnbieren — APPROVED** [review]

**Comment by johnbieren** [comment]
I wonder why the other team members aren't listed as reviewers. Shouldn't CODEOWNERS have added everyone? Maybe once someone approves, the others from the team are removed?

**Comment by mmalina** [comment]
> I wonder why the other team members aren't listed as reviewers. Shouldn't CODEOWNERS have added everyone? Maybe once someone approves, the others from the team are removed?

Not sure how this works. But I created a test PR and it added the whole team as reviewers: https://github.com/redhat-appstudio/release-service-catalog/pull/352

But I guess once one of us actively reviews it, the group is removed maybe?

**Review by happybhati — APPROVED** [review]

**Pull request merged**
